### DISCUSSION TITLE
backgroundColor: Can I  change the backgroundcolor of canvas panel? How?

### DISCUSSION THREAD

**Discussion by Angry-Sparrow** [storybookjs/storybook#14975] [Help]
I set the `defaultViewport` to `iphone6`.
The canvas is empty and the color is ugly. 
I want to change the color.
Is there any way？

![image](https://user-images.githubusercontent.com/48384080/118603109-1ce40a00-b7e6-11eb-91db-ec87ab4e422b.png)

**Comment by mayank99** [comment]
Not sure it would help in your case but we had a similar problem and ended up just adding a css rule in `preview-head.html`

**Reply by Angry-Sparrow** [reply]
Thanks. I will try.
